dashboard/app/assets/javascripts/levels/contract_match.js.jsx

```diff
@@ -1,5 +1,42 @@
 /** TODO(bjordan): Move this all into StudioApps */
```

Comment:
Removing comment, this seems somewhat out of date. If this happened it'd be part of a broader migration / not practical as a one-off